app/views/files/index.slim

```diff
@@ -19,6 +19,8 @@ ul
 					input type="hidden" name="_method" value="DELETE"
 					button type="submit" Delete
 
+				a href="#{file.route}?download=true" Download
+
```

Comment:
What would this mean for directories? For example I have the folder `uploads`. Should the path to view uploads be `/:site/files/uploads/edit`?  If so, what should happen when you visite `/:site/files/uploads`?

We could have directories behave as they currently do (no edit required) and have the edit route only for files, but that seems a little strange to me.